chrome/content/coniks/coniks.js

```diff
@@ -0,0 +1,82 @@
+this.EXPORTED_SYMBOLS = ["coniks"];
+
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("resource:///modules/imServices.jsm");
+Cu.import("resource:///modules/imXPCOMUtils.jsm");
+Cu.import("resource:///modules/jsProtoHelper.jsm");
+Cu.import("chrome://otr/content/coniks/context.js");
+Cu.import("chrome://otr/content/coniks/policies.js");
+Cu.import("chrome://otr/content/helpers.js");
+
+function accountname(acc) {
+  let account = acc.normalizedName;
+  let protocol = acc.protocol.normalizedName;
+  return account+"@"+protocol;
+}
+
+var coniks = {
+  _isEnable: true,
+  get Enabled() { return coniks._isEnable; },
+
+  init: function() {
+    coniks.setConfig();
+    this._contexts = new Map();
```

Comment:
I made some changes in ba666d68ef59fbf75aa2af57a213ed5272f89df7. We can add more fields (e.g., TB) into `AccountContext` in the future. 

P.S. Sorry for not make the changes in separate commit. (I forgot it).